dmcontent/govuk_frontend.py

```diff
@@ -107,11 +107,30 @@ def from_question(
 def govuk_input(
     question: Question, data: Optional[dict] = None, errors: Optional[dict] = None, **kwargs
 ) -> dict:
-    """Create govukInput macro parameters from a text question"""
+    """Create govukInput macro parameters from a text or number question"""
 
     params = _params(question, data, errors)
     params["classes"] = "app-text-input--height-compatible"
 
+    if question.type == "number":
+        params["classes"] += " govuk-input--width-5"
+        params["spellcheck"] = False
```

Comment:
👍 

Don't think there's many instances of us using text inputs for anything which would need a spellcheck.

Perhaps we should add it to the textareas though...

Reply:
I just copied from the Design System examples haha https://design-system.service.gov.uk/components/text-input/#numbers

Good idea adding `spellcheck=True` to textareas though!

Reply:
I've added that to the `govuk_character_count` function